airflow/operators/trigger_dagrun.py

```diff
@@ -199,7 +199,8 @@ def execute(self, context: Context):
 
         except DagRunAlreadyExists as e:
             if self.reset_dag_run:
-                self.log.info("Clearing %s on %s", self.trigger_dag_id, parsed_logical_date)
+                dag_run = e.dag_run
+                self.log.info("Clearing %s on %s", self.trigger_dag_id, dag_run.logical_date)
```

Comment:
Also fixing the log message when a task is configured to reset existing DAG runs.